main.py

```diff
@@ -1,4 +1,13 @@
 import requests 
+from bs4 import BeautifulSoup
 
 players_url =  "https://proclubshead.com/24/club-squad/gen5-179156/" 
-data = requests.get(players_url) 
\ No newline at end of file
+data = requests.get(players_url) 
+
+soup = BeautifulSoup(data.text) 
+members = soup.select('div.bg-gradient')[0] 
+links = members.find_all('a') 
+links = (l.get("href") for l in links) 
+links = [l for l in links if '/club-player/']
```

Comment:
There shouldn't be 3 variables with the same name, makes the code confusing. I think you can make each variable more descriptive based on what each is doing